jwst/assign_wcs/niriss.py

```diff
@@ -135,29 +135,25 @@ def niriss_soss(input_model, reference_files):
     try:
         # We'd like to open this file as a DataModel, so we can consolidate
         # the S3 URI handling to one place.  The S3-related code here can
-        # be removed once that has been changed.
+        # be removed once we have a specwcs DataModel subclass.
         if s3_utils.is_s3_uri(reference_files['specwcs']):
-            wl = asdf.open(s3_utils.get_object(reference_files['specwcs']))
+            uri = s3_utils.get_object(reference_files['specwcs'])
```

Comment:
This accepts a URI and returns an `io.BytesIO` (not another URI)

Reply:
I changed the variable name to make this clear.